src/algorithms/crypto/randomness.md

```diff
@@ -21,58 +34,25 @@ Further, we define Domain Separation Tags with which we prepend random inputs wh
 All randomness used in the protocol must be generated in conjunction with a unique DST, as well as 
 certain {{<sref crypto_signatures>}} and {{<sref vrf>}} usage.
 
-## Drawing tickets for randomness from the chain
-
-Tickets are used as a source of on-chain randomness, generated with each new block created (see {{<sref tickets>}}).
-
-A ticket is drawn from the chain for randomness as follows, for a given epoch `n`, and ticket sought at epoch `e`:
-```text
-RandomnessSeedAtEpoch(e):
-    While ticket is not set:
-        Set wantedTipsetHeight = e
-        if wantedTipsetHeight <= genesis:
-            Set ticket = genesis ticket
-        else if blocks were mined at wantedTipsetHeight:
-            ReferenceTipset = TipsetAtHeight(wantedTipsetHeight)
-            Set ticket = minTicket in ReferenceTipset
-        If no blocks were mined at wantedTipsetHeight:
-            wantedTipsetHeight--
-            (Repeat)
-    return ticket.Digest()
-```
-
-In plain language, this means:
-
-- Choose the smallest ticket in the Tipset if it contains multiple blocks.
-- When sampling a ticket from an epoch with no blocks, draw the min ticket from the prior epoch with blocks
-
-This ticket is then combined with a Domain Separation Tag, the round number sought and appropriate entropy to form randomness for various uses in the protocol.
-
-See the `RandomnessSeedAtEpoch` method below:
-{{< readfile file="../struct/chain/chain.go" code="true" lang="go" >}}
-
 ## Forming Randomness Seeds
 
-The drawn ticket digest is combined with a few elements to make up randomness for use as part of the protocol.
+The beacon entry is combined with a few elements for use as part of the protocol as follows:
 
 - a DST (domain separation tag)
     - Different uses of randomness are distinguished by this type of personalization which ensures that randomness used for different purposes will not conflict with randomness used elsewhere in the protocol
 - the epoch number, ensuring
-    - liveness for leader election -- in the case of null rounds, the new epoch number will output new randomness for LE
-    - distinct values for randomness sought before genesis -- where the genesis ticket will be returned
-    - For instance, if in epoch `curr`, a miner wants randomness from `lookback` epochs back where `curr - lookback <= genesis`, the ticket randomness drawn would be based on `genesisTicket.digest` where the `genesisTicket` is the randomness included in the genesis block. Using the epoch as part of randomness composition ensures that randomness drawn at various epochs prior to genesis has different values.
-- other entropy,
-    - ensuring that randomness is modified as needed by other context-dependent entropy (e.g. a miner address if we want the randomness to be different for each miner).
+    - liveness for leader election -- in the case no one is elected in a round and no new drand entry has appeared, the new epoch number will output new randomness for LE
```

Comment:
1. "drand" -> "beacon".
2. I am not sure I agree with the fact that liveness is guaranteed in the absence of the beacon in the current version of the protocol.

Reply:
where do you see that second claim made? The above statement only says that if the frequency of the beacon is less than Fil's, we still have liveness. I'll detail a bit to avoid confusion